atomix/cluster/src/test/java/io/atomix/raft/ControllableRaftContexts.java

```diff
@@ -51,6 +51,7 @@
 import java.util.Collections;
```

Comment:
This is the only change, which makes me a bit worried. Please take a look at whether we are fine with this, we need to find a way to remove the object from the listener.